src/components/sbb-time-input/sbb-time-input.e2e.ts

```diff
@@ -34,6 +34,20 @@ describe('sbb-time-input', () => {
     expect(await input.getProperty('value')).toEqual('11:00');
   });
 
+  it('should watch for value changes and interpret valid values', async () => {
+    element.setProperty('value', ':00');
+    await page.waitForChanges();
+    const input = await page.find('sbb-time-input >>> input');
+    expect(await input.getProperty('value')).toEqual('00:00');
+  });
+
+  it('should watch for value changes and clear invalid values', async () => {
+    element.setProperty('value', ':');
+    await page.waitForChanges();
+    const input = await page.find('sbb-time-input >>> input');
+    expect(await input.getProperty('value')).toEqual('');
+  });
+
```

Comment:
could also the Backspace behavior be tested somehow? like with await element.press('Backspace');

Reply:
Since the bug was not strictly related to the Backspace button, but rather to the regex kicking in whenever the input had a value like ":mm", I implemented a test similar to the one you suggested, but using the Delete button instead (just to avoid handling the cursor position by hand)